services.py

```diff
@@ -104,3 +105,24 @@ def update_or_create_criterion(data, criterion=None):
         criterion.deactivate()
 
     return criterion.save()
+
+
+forms = [
+    'feedback',
+    'report_missing_info',
+    'partner_with_freefrom',
+    'build_collective_survivor_power',
+    'policy_ideas',
+]
+
+
+def submit_form_to_google(form, data):
+    if form not in forms:
+        raise ValueError(strings.form_not_found)
```

Comment:
The app script will also return an error if we give it an invalid form name, but I added this check in so we could provide a more descriptive status code.